include/psp2kern/kernel/cpu.h

```diff
@@ -253,23 +253,23 @@ int ksceKernelCpuSpinLockIrqSave(int *addr);
  */
 int ksceKernelCpuSpinLockIrqRestore(int *addr, int flags);
 
-typedef enum SceCpuWaitSyncCore {
-	SCE_CPU_WAIT_CORE_0 = 1,
-	SCE_CPU_WAIT_CORE_1 = 2,
-	SCE_CPU_WAIT_CORE_2 = 3,
-	SCE_CPU_WAIT_CORE_3 = 0
-} SceCpuSyncWaitCore;
-
-typedef struct SceCpuCoreSyncCtx {
+typedef enum SceCorelockCore {
+	SCE_CORELOCK_CORE0 = 1,
+	SCE_CORELOCK_CORE1 = 2,
+	SCE_CORELOCK_CORE2 = 3,
+	SCE_CORELOCK_CORE3 = 0
+} SceCorelockCore;
```

Comment:
how about it just change to `SceCPUCore` & `SCE_CPU_CORE_0`?

Reply:
I forgot to add it to the description, but these functions are bruteforced names.